ts-obs/cmd/helmInstall.go

```diff
@@ -0,0 +1,111 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+const DEVEL = true
+const REPO_LOCATION = "https://charts.timescale.com"
+
+// helmInstallCmd represents the helm install command
+var helmInstallCmd = &cobra.Command{
+	Use:   "install",
+	Short: "Installs Timescale Observability",
+	Args:  cobra.ExactArgs(0),
+	RunE:  helmInstall,
+}
+
+func init() {
+	helmCmd.AddCommand(helmInstallCmd)
+	helmInstallCmd.Flags().StringP("filename", "f", "", "YAML configuration file to load")
+}
+
+func helmInstall(cmd *cobra.Command, args []string) error {
+	var err error
+
+	var name string
+	name, err = cmd.Flags().GetString("name")
+	if err != nil {
+		return fmt.Errorf("could not install Timescale Observability: %w", err)
+	}
+
+	var file string
+	file, err = cmd.Flags().GetString("filename")
+	if err != nil {
+		return fmt.Errorf("could not install Timescale Observability: %w", err)
+	}
+
+	var namespace string
+	namespace, err = cmd.Flags().GetString("namespace")
+	if err != nil {
+		return fmt.Errorf("could not install Timescale Observability: %w", err)
+	}
+
+	w := io.Writer(os.Stdout)
+
+	addchart := exec.Command("helm", "repo", "add", "timescale", REPO_LOCATION)
+
+	addchart.Stdout = w
+	addchart.Stderr = w
+	fmt.Println("Adding Timescale Helm Repository")
+	err = addchart.Run()
+	if err != nil {
+		return fmt.Errorf("could not install Timescale Observability: %w", err)
+	}
+
+	update := exec.Command("helm", "repo", "update")
+
+	update.Stdout = w
+	update.Stderr = w
+	fmt.Println("Fetching updates from repository")
+	err = update.Run()
+	if err != nil {
+		return fmt.Errorf("could not install Timescale Observability: %w", err)
+	}
+
+	var install *exec.Cmd
+	if DEVEL {
+		if file == "" {
+			install = exec.Command("helm", "install", name, "timescale/timescale-observability", "--devel")
+		} else {
+			install = exec.Command("helm", "upgrade", "--install", name, "--values", file, "timescale/timescale-observability", "--devel")
```

Comment:
do we want ts-obs install to upgrade if the release exists. I think not. I'd rather add a ts-obs upgrade command that installs if things don't exist than the other way around.